doc/how_to/server/websockets.md

```diff
@@ -0,0 +1,122 @@
+# Configuring WebSocket Communication Settings
+
+This guide provides comprehensive instructions for configuring bidirectional WebSocket communication in Panel/Bokeh/Tornado/Jupyter/Kubernetes environments to handle larger data transfers effectively.
+
+Out of the box, you can reliably handle communication up to approximately 10 MB. This covers common use cases including visualizing tabular data and plots and uploading smaller files. But for example for audio and video use cases you may quickly run into issues.
+
+## Overview
+
+When building applications that handle significant data transfers (file uploads/ downloads, large visualizations, audio and video), the default WebSocket communication settings may become a bottleneck. Understanding and configuring these settings is crucial for production deployments.
+
+## Understanding WebSocket Communication Flow
+
+**The journey of data from client to server:**
+
+1. **Browser processing** - Data is loaded into browser memory
+2. **Message packaging** - The data is packaged into JSON messages. This might sometimes increase or decrease the size depending on the serialization method applied.
+3. **WebSocket transmission** - Data travels through [WebSocket](https://en.wikipedia.org/wiki/WebSocket) connections via [Tornado](https://www.tornadoweb.org/en/stable/).
+4. **Backend processing** - Your Python application receives and unpacks the JSON Messages.
+
+## Technical Constraints and Default Limits
+
+Understanding the various layers that impose size limitations helps you configure your environment appropriately:
+
+| Component | Default Limit | Description |
+|-----------|---------------|-------------|
+| Browser | Several GB | Rarely the bottleneck |
+| WebSocket Message Size | 20 MB | Tornado/Bokeh default setting |
+| Buffer Size | 100 MB | Tornado default setting |
+| Network/Infrastructure | Varies | Kubernetes, load balancers, proxies may impose additional limits |
+
+## Configuration Guidelines
+
+### Panel/Bokeh Server Deployments
+
+For production applications using **`panel serve`**, configure larger limits at application startup:
+
+WHAT SHOULD BE DONE HERE?
```

Comment:
Yes. What should be done here? The problem is that there are no `pn.config` parameters you can set. You can set `--websocket-max-message-size`. I see no way of setting `http_server_kwargs`.

I need help from someone to understand how to set these parameters.